### ISSUE TITLE
[node10] Test failure: "up command delivers modified.."

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-cli#608]
**To Reproduce**
1. use node.js 10
2. run `npm test -- -fgrep "up command delivers modified sources and delivers correct response."`
3. see that the test fails:

```
1) up command delivers modified sources and delivers correct response.

  0 passing (10s)
  1 failing

  1) Integration test for up command
       up command delivers modified sources and delivers correct response.:
     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

**Version:**
```
$ hlx --version && node --version
0.13.6-pre.15
v10.15.1
```

**Additional context**
the exact same works with node 8

**Comment by stefan-guggisberg** [comment]
**TLDR:** Not a bug. Make sure to `rm -rf node_modules && npm i` when switching node versions.

The above described test failure can be reproduced as follows:
```
nvm use 8
rm -rf node_modules
npm i 
npm test
# ok
nvm use 10
npm test
# 1 failure
```

The problem is the native `fsevents` addon which comes with node version-specific `fse.node` shared objects. `fse.node` installed on node 8 doesn't work on node 10. It needs be installed/built on node 10. 

The following sequence works:

```
nvm use 8
rm -rf node_modules
npm i 
npm test
# ok
nvm use 10
rm -rf node_modules
npm i 
npm test
# ok
```
